<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shoppingW</class>
 <widget class="QDialog" name="shoppingW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_addItem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addStore">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Store</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Main</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbutton_UpdateStore">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Store</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UpdateItem">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ViewItems">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View All Items</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ViewStores">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View All Stores</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="storeNameEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="storeTypeEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemTypeEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemPriceEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>storeName</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>storeType</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>itemType</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>itemPrice</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
